go.mod

```diff
@@ -120,7 +120,7 @@ require (
 	k8s.io/sample-apiserver v0.0.0
 	k8s.io/system-validators v1.8.0
 	k8s.io/utils v0.0.0-20230726121419-3b25d923346b
-	sigs.k8s.io/knftables v0.0.14
+	sigs.k8s.io/knftables v0.0.16-0.20240228165918-78f68d9b296e
```

Comment:
So it looks like we're good to tag this release then? (Also, this commit should be before the commit that depends on it, not after.)

Reply:
ok, `v0.0.16` is tagged